src/index.js

```diff
@@ -171,8 +171,16 @@ export default {
     // Required - must be implemented
     // Browser control
     async openBrowser (id, pageUrl, browserName) {
-        var capabilities = { ...this._generateBasicCapabilities(browserName), ...this._getAdditionalCapabilities() };
-        var connector    = await this._getConnector();
+        var capabilities = {
+            ...this._generateBasicCapabilities(browserName),
+            ...this._getAdditionalCapabilities()
+        };
+        const localIdentifier = process.env['BROWSERSTACK_LOCAL_ID'];
+
+        let connector = null;
+
+        if (!localIdentifier)
```

Comment:
If you don't create a connector, you should avoid disposing `null` here:
https://github.com/DevExpress/testcafe-browser-provider-browserstack/blob/master/src/index.js#L217

Reply:
It will be resolved to null only. Is there any harm in this approach?

Reply:
Nevermind, I forgot that the check for `null` is implemented already: https://github.com/DevExpress/testcafe-browser-provider-browserstack/blob/master/src/index.js#L47